src/llmq/quorums_chainlocks.cpp

```diff
@@ -214,12 +214,13 @@ void CChainLocksHandler::CheckActiveState()
     LOCK(cs);
     bool oldIsEnforced = isEnforced;
     isSporkActive = sporkManager.IsSporkActive(SPORK_19_CHAINLOCKS_ENABLED);
-    isEnforced = fDIP0008Active && isSporkActive;
-
+    // TODO remove this after DIP8 is active
+    bool fEnforcedBySpork = Params().NetworkIDString() == CBaseChainParams::TESTNET && (sporkManager.GetSporkValue(SPORK_19_CHAINLOCKS_ENABLED) == 1);
+    isEnforced = (fDIP0008Active && isSporkActive) || fEnforcedBySpork;    isEnforced = fDIP0008Active && isSporkActive;
```

Comment:
```suggestion
    isEnforced = (fDIP0008Active && isSporkActive) || fEnforcedBySpork;
```

Reply:
Ouch...thanks for spotting this one, that would have led to some waste of time through debugging...